ui/app/mainui/SplashScreen.qml

```diff
@@ -17,24 +17,17 @@ Item {
         }
     }
 
-    Image {
-        width: 150
-        height: 150
-        fillMode: Image.PreserveAspectFit
+    AnimatedImage {
+        id: splashLogo
         anchors.centerIn: parent
-        source: Style.svg("status-logo-circle")
+        source: Style.gif("status_splash")
     }
 
-//    LottieAnimation {
-//        anchors.centerIn: parent
-//        autoPlay: false
-//        loops: LottieAnimation.Infinite
-//        quality: LottieAnimation.MediumQuality
-//        source: Style.lottie("status_splash")
-//        onStatusChanged: {
-//            if (status === LottieAnimation.Ready) {
-//                start();
-//            }
-//        }
-//    }
+    // LottieAnimation {
```

Comment:
Do we want to keep this ?
If yes, then it would be good to add the comment why it is here.